app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionBlockModal/serviceVersionBlockModal.jsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalLayout } from 'componentLibrary/modal';
+import classNames from 'classnames/bind';
+import { BigButton } from 'components/buttons/bigButton';
+import { FormattedMessage } from 'react-intl';
+import styles from './serviceVersionsBlockModal.scss';
+import { ServiceVersionsBlockWithTooltip } from '../serviceVersionBlockTooltip/serviceVersionsBlock';
+
+const cx = classNames.bind(styles);
+
+const FooterNode = ({ hideModal }) => (
+  <div className={cx('modal-footer-node')}>
+    <BigButton roundedCorners color="organish" onClick={hideModal}>
+      <FormattedMessage id={'Common.close'} defaultMessage={'Close'} />
```

Comment:
Looks like you override standard key here. You can use formatMessage from useIntl hook `formatMessage(COMMON_LOCALE_KEYS.CLOSE)`
instead FormattedMessage component